internal/handlers/common/update.go

```diff
@@ -368,6 +375,43 @@ func processMaxFieldExpression(doc *types.Document, updateV any) (bool, error) {
 	return changed, nil
 }
 
+// processMinFieldExpression changes document according to $min operator.
+// If the document was changed it returns true.
+func processMinFieldExpression(doc *types.Document, updateV any) (bool, error) {
+	minExpression := updateV.(*types.Document)
+	minExpression = minExpression.SortFieldsByKey()
```

Comment:
Sorting _update_ document fields by key to ensure _MultipleQueriesSorted_ test passes.